governance/votes/Executive vote - July 4, 2024.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Template - [Executive Vote] Out-of-Schedule Executive Vote - Approve LITE-PSM-USDC DAO Resolution - July 04, 2024
+summary: Approving a DAO Resolution pertaining to the existing Coinbase Custody arrangement (RWA014-A) and the new Lightweight Peg Stability Moodule (LITE-PSM-A). 
+date: 2024-07-04T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Out-of-Schedule Executive Vote - Approve LITE-PSM-USDC DAO Resolution - July 04, 2024
+
+The Governance Facilitators and Dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The "Approve LITE-PSM-USDC" DAO Resolution will be approved.
+
```

Comment:
As per the template, we're missing the statement:

**Voting for this executive proposal will place your MKR in support of the actions outlined above.**